### DISCUSSION TITLE
Start up dev app with Referrer info

### DISCUSSION THREAD

**Discussion by timenengwerda** [ionic-team/capacitor#6183] [Q&A]
I have an app that is heavily reliant on the referrer data the app gets when the user installs the app. It gets some data from the referrer and saves it in localStorage.

Now this is a complete PEST to test on a local Emulator because it doesn't have the same referrer data than when you would download the app from the store. So for now I have to release an internal test to the Google Play store everytime I want to test out some referrer stuff and you can imagine the hassle of that.

Now I've found a way you can send a broadcast referrer but I can't find a proper way to catch that referrer(All capacitor/ionic/cordova plugins are obsolete/deprecated and not working anymore).

I'm Using Vue3+Vite to run my server.

Now my question in short is: When I run `npm run cap sync && npm run open:android && vite` is there any way possible I can load up my emulator with a certain set of referrer data?

**Comment by Thomas196-Sharon** [comment]
Hey Sir, Tomorrow what time you will be available so that we can connect virtually to discuss your requirements?
My email- is sharon@plaxonic.com
